overlays/overlays_kustomization_template

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../baseDir/
+
+patchesStrategicMerge:
+%s
```

Comment:
Could you leverage here Go Template? https://golang.org/pkg/text/template/ Template is a powerful language with a lot of primitives pretty robust to manipulate a file. This way is error-prone.

Reply:
sure, I will check and will push the respective change.